src/contexts/navigation.js

```diff
@@ -135,6 +135,10 @@ export function NavigationProvider({ children }) {
     let finalDate = dateParam ? dateParam : date;
     let finalSource = sourceParam ? sourceParam : source;
     setDataAll(finalDate, finalSource);
+
+    retrieveMeteors(date, source);
```

Comment:
why do we need to retrieve these data again here?

Reply:
I'll take this out